frontend/src/components/ui/copy-button.ts

```diff
@@ -61,15 +69,16 @@ export class CopyButton extends LitElement {
         @sl-after-hide=${this.stopProp}
       >
         <btrix-button
-          size="small"
+          size=${this.size}
           @click=${this.onClick}
           ?disabled=${!this.value && !this.getValue}
           class="inline"
-          raised
+          ?raised=${this.raised}
         >
           <sl-icon
             name=${this.isCopied ? "check-lg" : this.name ? this.name : "copy"}
             label=${msg("Copy to clipboard")}
+            class="size-[14px]"
```

Comment:
Does `text-small` work here? Should be set to 0.875rem / 14px

Reply:
Or, `size-3.5`

Reply:
No, `text-` sets font size, `size-` sets width and height

Reply:
Oh you're right! I forgot there are 0.5 increments in the lower values

Reply:
I think you might need both to be safe, since `sl-icon` will inherit font size. And/or we should explicitly set font size in `button`.

Reply:
Ahh, sure, being set in the button is fair, just a min